instana/__init__.py

```diff
@@ -55,9 +55,6 @@ def load(module):
 #
 internal_tracer = InstanaTracer()
 
-# Set ourselves as the tracer.
-opentracing.tracer = internal_tracer
-
```

Comment:
I'm curious as to the reason for this removal.  This allows OpenTracing plugability.  So existing calls in an application to `opentracing.tracer.start_span` etc just automatically work when the instana package is used.

Reply:
I know this is WIP so I don't mean to overload :-)

Reply:
In other OT libraries, it's more common (and I would argue less-surprising) to not automatically modify another package's variables. For example, with https://github.com/lightstep/lightstep-tracer-python, to set the opentracing.tracer, they recommend doing this:

```
  opentracing.tracer = lightstep.Tracer(
    component_name='your_microservice_name',
    access_token='{your_access_token}')
```
Although, yes, it is setting opentracing.tracer, it's giving the user of the library the option to set it when they want and not doing it automatically.

In probably MOST cases, this works fine, but I'm trying to test tracers to multiple systems. One way to do this is to build a multi-tracer, which calls the start_span (or any other methods) on an array of tracers. It's not quite that simple, but that's the basic idea.